client/helpers.go

```diff
@@ -26,13 +27,16 @@ func getRemoteStore(baseURL, gun string, rt http.RoundTripper) (store.RemoteStor
 
 func applyChangelist(repo *tuf.TufRepo, cl changelist.Changelist) error {
 	changes := cl.List()
-	var err error
+	logrus.Debugf("applying %d changes", len(changes))
 	for _, c := range changes {
-		if c.Scope() == "targets" {
-			applyTargetsChange(repo, c)
-		}
-		if err != nil {
-			return err
+		switch c.Scope() {
+		case changelist.ScopeTargets:
+			err := applyTargetsChange(repo, c)
+			if err != nil {
+				return err
+			}
+		default:
+			logrus.Debug("scope not supported: ", c.Scope())
```

Comment:
We weren't before either. Unsupported changes should never be in the changelist so we skip them.